src/plugins/gui/scenes/SepaTransferScene.tsx

```diff
@@ -0,0 +1,180 @@
+import { useHandler } from '../../../hooks/useHandler'
+import s from '../../../locales/strings'
+import { RouteProp } from '../../../types/routerTypes'
+
+interface SepaDisplayGroup {
+  groupTitle: string
+  items: Array<{ label: string; value?: string }>
+}
+
+interface Props {
+  route: RouteProp<'guiPluginSepaTransfer'>
+}
+
+export const SepaTransferScene = React.memo((props: Props) => {
+  const theme = useTheme()
+  const styles = getStyles(theme)
+  const { route } = props
+  // TODO: headerIconUri
+  const { headerTitle, transferInfo, promptMessage, onDone } = route.params
+
+  const displayData: SepaDisplayGroup[] = React.useMemo(() => {
+    const { input, output, paymentDetails } = transferInfo
+    return [
+      {
+        groupTitle: s.strings.quote_input_title,
+        items: [
+          {
+            label: s.strings.string_amount,
+            value: input.amount
+          },
+          {
+            label: s.strings.input_output_currency,
+            value: input.currency
+          }
+        ]
+      },
+      {
+        groupTitle: s.strings.quote_output_title,
+        items: [
+          {
+            label: s.strings.string_amount,
+            value: output.amount
+          },
+          {
+            label: s.strings.input_output_currency,
+            value: output.currency
+          },
+          {
+            label: s.strings.request_qr_your_wallet_address,
+            value: output.walletAddress
+          }
+        ]
+      },
+      {
+        groupTitle: s.strings.payment_details,
+        items: [
+          {
+            label: s.strings.transaction_details_exchange_order_id,
+            value: paymentDetails.id
+          },
+          {
+            label: s.strings.iban,
+            value: paymentDetails.iban
+          },
+          {
+            label: s.strings.swift_bic,
+            value: paymentDetails.swiftBic
+          },
+          {
+            label: s.strings.transaction_details_recipient,
+            value: paymentDetails.recipient
+          },
+          {
+            label: s.strings.reference,
+            value: paymentDetails.reference
+          }
+        ]
+      }
+    ]
+  }, [transferInfo])
+
+  const handleCopyPress = useHandler((value: string) => {
+    Clipboard.setString(value)
+  })
+
+  const handleDone = useHandler(async () => {
+    await onDone()
+  })
+
+  const renderCopyButton = (value: string) => {
+    return (
+      <TouchableOpacity onPress={() => handleCopyPress(value)}>
+        <Fontello name="Copy-icon" size={theme.rem(1)} color={theme.iconTappable} />
+      </TouchableOpacity>
+    )
+  }
+
+  const renderGroups = () =>
+    displayData.map(group => (
+      <View style={styles.groupContainer} key={group.groupTitle}>
+        <EdgeText style={styles.groupTitle}>{group.groupTitle}</EdgeText>
+        {group.items.map(item => (
+          <View style={styles.textRow} key={item.label}>
```

Comment:
This looks VERY similar to a Tile. 

For another PR: Refactor to use Tile's which automatically have optional Copy buttons